scripts/test/hyriseConsole_test.py

```diff
@@ -44,9 +44,13 @@ def main():
 	console.expect("Execution info:")
 
 	# Test TPCH generation
-	console.sendline("generate_tpch 0.01")
+	console.sendline("generate_tpch     0.01   7")
 	console.expect("Generating tables done")
 
+	# Test correct chunk size
+	console.sendline("print nation")
+	console.expect("=== Chunk 3 ===")
```

Comment:
Explains this a bit:

* table nation has 25 rows and does not scale with the SF
* chunk-size 7 means a minimum/maximum of three chunks